code/modules/awaymissions/maploader/writer.dm

```diff
@@ -5,28 +5,27 @@
 #define DMM_IGNORE_PLAYERS 16
 #define DMM_IGNORE_MOBS 24
 #define DMM_USE_JSON 32
-/dmm_suite
-	var/quote = "\""
-	var/list/letter_digits = list(
-			"a","b","c","d","e",
-			"f","g","h","i","j",
-			"k","l","m","n","o",
-			"p","q","r","s","t",
-			"u","v","w","x","y",
-			"z",
-			"A","B","C","D","E",
-			"F","G","H","I","J",
-			"K","L","M","N","O",
-			"P","Q","R","S","T",
-			"U","V","W","X","Y",
-			"Z"
-			)
-
-/dmm_suite/save_map(var/turf/t1 as turf, var/turf/t2 as turf, var/map_name as text, var/flags as num)
-	//Check for illegal characters in file name... in a cheap way.
-	if(!((ckeyEx(map_name)==map_name) && ckeyEx(map_name)))
+/datum/dmm_suite
+	var/static/list/letter_digits = list(
```

Comment:
Shouldnt this be in `dmm_suite.dm`

Reply:
You're right, in my mind I thought it'd make more sense to put this in `writer.dmm` because it's exclusively used there but I think for consistency all static stuff should be in the main file along with `var/static/quote` and the regexes